tests/ui/unit/filtersSpec.js

```diff
@@ -1,19 +1,245 @@
+    beforeEach(inject(function(_$filter_) {
+        $filter = _$filter_;
+    }));
+
+    it('escapes some html symbols', function() {
+        var escapeHTML = $filter('escapeHTML');
+        expect(escapeHTML('<This \'is the \"worst>')).toEqual('&lt;This &#39;is the &quot;worst&gt;');
+    });
+});
+
+describe('getRevisionUrl filter', function() {
+    var $filter;
+    beforeEach(module('treeherder'));
+    beforeEach(inject(function(_$filter_) {
+        $filter = _$filter_;
+    }));
+
+    it('escapes some html symbols', function() {
+        var getRevisionUrl = $filter('getRevisionUrl');
+        expect(getRevisionUrl('1234567890ab', 'mozilla-inbound'))
+          .toEqual('/#/jobs?repo=mozilla-inbound&revision=1234567890ab');
+    });
+});
+
+describe('showOrHide filter', function() {
+    var $filter;
+    beforeEach(module('treeherder'));
+    beforeEach(inject(function(_$filter_) {
+        $filter = _$filter_;
+    }));
+
+    it('says show', function() {
+        var showOrHide = $filter('showOrHide');
+        expect(showOrHide(' me', true)).toEqual('show me');
+    });
+
+    it('says hide', function() {
+        var showOrHide = $filter('showOrHide');
+        expect(showOrHide(' me', false)).toEqual('hide me');
+    });
+});
+
+describe('stripHtml filter', function() {
+    var $filter;
+    beforeEach(module('treeherder'));
+    beforeEach(inject(function(_$filter_) {
+        $filter = _$filter_;
+    }));
+
+    it('deletes html tags', function() {
+        var stripHtml = $filter('stripHtml');
+        expect(stripHtml('My <html is> deleted')).toEqual('My  deleted');
+    });
+});
+
+describe('linkifyClassifications filter', function() {
+    var $filter;
+    beforeEach(module('treeherder'));
+    beforeEach(inject(function(_$filter_) {
+        $filter = _$filter_;
+    }));
+
+    it('linkifies classifications', function() {
+        var linkifyClassifications = $filter('linkifyClassifications');
+        // This keeps saying "repo is null"...
+        //expect(linkifyClassifications('This 1234567890ab is linked', 'try')).toEqual('');
+    });
+});
+
+describe('encodeURIComponent filter', function() {
+    var $filter;
+    beforeEach(module('treeherder'));
+    beforeEach(inject(function(_$filter_) {
+        $filter = _$filter_;
+    }));
+
+    it('encodes uri components', function() {
+        var encodeURIComponent = $filter('encodeURIComponent');
+        expect(encodeURIComponent('this/is.a?URI#Component')).toEqual('this%2Fis.a%3FURI%23Component');
+    });
+});
+
+/*
+ * I don't know what the perfherder filters are supposed to do, but I can add stubs for them
+describe('alertStatus filter', function() {
+    var $filter;
+    beforeEach(module('treeherder'));
+    beforeEach(inject(function(_$filter_) {
+        $filter = _$filter_;
+    }));
+
+    it('does some more things', function() {
+        var alertStatus = $filter('alertStatus');
+        expect(alertStatus('')).toEqual('');
+    });
+});
```

Comment:
I believe the alert status filter isn't actually used, so you can just skip it. :)